backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -738,6 +738,7 @@ void SurfaceSdlGraphicsManager::getDefaultResolution(uint &w, uint &h) {
 	SDL_Rect const* const*availableModes = SDL_ListModes(&p, SDL_FULLSCREEN|SDL_HWSURFACE);
 	w = availableModes[0]->w;
 	h = availableModes[0]->h;
+	if (h > w) h /= 2;
```

Comment:
Could you add a comment here?

Alternatively, does _window-> getDesktopResolution() give expected results for both RS90 and RG99?

Reply:
@ccawley2011 comment added. As the maintainer of the OD port I feel this is getting too close to the freeze to experiment alternatives now, I'd rather have this PR expedited to unbreak the rg99. Then we can have a discussion about how to handle things better for the next one